metadash/models/collections/attribute_mapped_list_collection.py

```diff
@@ -141,7 +141,12 @@ def remove(self, value, _sa_initiator=None):
 
     @collection.internally_instrumented
     def __setitem__(self, key, value, appender=None):
-        appender = appender or collection_adapter(self).fire_append_event
+        c_inst = collection_adapter(self)
+        if not appender:
+            # Avoid AttributeError: 'NoneType' object has no attribute
+            # 'fire_append_event'
+            if c_inst:
+                appender = c_inst.fire_append_event
```

Comment:
That's looks a tricky problem, let me see if I can reproduce with unittest before merge.